vllm/entrypoints/openai/rpc/server.py

```diff
@@ -0,0 +1,216 @@
+import asyncio
+import signal
+from typing import Any, Coroutine
+
+import cloudpickle
```

Comment:
need this in requirements txt

Reply:
I can explictily add it, but it seems to be installed already